angular/src/app/services/wallet-manager.ts

```diff
@@ -67,6 +67,11 @@ export class WalletManager {
         return (this.secure.get(this.activeWalletId) != null);
     }
 
+    validateMnemonic( mnemonic: string) {
+      return of(bip39.validateMnemonic(mnemonic)).pipe(
+        delay(1000)
```

Comment:
Why is there a delay here? It makes the UI behave a bit weird as validation can take up to 1 second?

Reply:
I put it to show to the user. I better clean it. Because I do not show an error to the user.

Reply:
The input field becomes red with warning, we can add additional error message below the textbox later. This delay simply delays the showing of the red error.

Reply:
Yes . I picked it up. I do not think the error message is necessary. The same red line is enough